doc/rados/operations/pools.rst

```diff
@@ -574,6 +574,14 @@ You may set values for the following keys:
    :Type: Integer
    :Default: ``0``
 
+.. _read_ratio:
+
+.. describe:: read_ratio
+
+   :Description: Sets the percent of read IOs out of all IOs for a pool. Applicable to replicated pools only. This value is used to improve read balancing when OSDs have different weights.
+
+   :Type: Integer
+   :Default: ``70``
```

Comment:
Change this; default is 0 (unset)

Reply:
@ljflores Shall I make these changes?

Reply:
@zdover23 no need to make these changes. I'll get your final review when the doc updates are ready, but it needs work from me and Josh.

Reply:
@ljflores, Roger that. I'll add this to my to-do list, two weeks from now.